src/wayland-egldisplay.c

```diff
@@ -730,7 +730,7 @@ static EGLBoolean terminateDisplay(WlEglDisplay *display, EGLBoolean globalTeard
     wlEglDestroyAllSurfaces(display);
 
     wlEglDestroyFormatSet(&display->formatSet);
```

Comment:
nit: Can you please remove this call to `wlEglDestroyFormatSet`? It predates your change but it's not needed as  `wlEglDestroyFeedback` will reset the tranches and call this anyway.

Reply:
Are you sure about this? To me it looks like this is a completely independent format set. It is used if the dmabuf version is <4.

Reply:
Ah good point. I think this should be moved to stay next to `wlEglDestroyFeedback` for consistency.